ansible/roles/installation/steamvr/tasks/main.yml

```diff
@@ -0,0 +1,97 @@
+---
+# SteamVR installation
+- name: Update package lists
+  apt: update_cache=yes
+  become: yes
+
+- name: Intall needed libs for steamvr
+  apt: name={{ item }} install_recommends=no
+  become: yes
+  with_items:
+    - curl
+    - ca-certificates
+    - lib32gcc1
+    - binutils
+    - terminator
+    - xdg-utils
+    - software-properties-common
+    - sudo
+    - libvulkan1
+    - usbutils
+    - libcap2-bin
+
+- name: Update and upgrade apt packages
+  become: true
+  apt:
+    upgrade: dist
+    update_cache: yes
+
+- name: Check if libudev0 is installed
```

Comment:
Please use this module https://docs.ansible.com/ansible/devel/modules/package_facts_module.html
As it is done in this example https://stackoverflow.com/a/52544520